javascript-loops-practice/filter-out-strings.js

```diff
@@ -3,7 +3,6 @@ function filterOutStrings(values) {
   var newArr = [];
   for (let i = 0; i < values.length; i++) {
     if (typeof values[i] === 'string') {
-      continue;
     } else {
       newArr.push(values[i]);
     }
```

Comment:
Here is what you want to do:
```
for (var i = 0; i < values.length; i++) {
    if (typeof values[i] !== 'string') {
      newArr.push(values[i]);
    }
}
```

Reply:
I see how that works!
I didn't think about changing my pseudocode.
Thank you, learned a lot!